src/aws/buildConfig.ts

```diff
@@ -0,0 +1,40 @@
+import { Schema } from "../config/schema.js";
+import type { SecretStored } from "./index.js";
+import { getItems, getSecrets } from "./index.js";
+
+export async function buildConfig(): Promise<Schema> {
+  const secretStored: SecretStored = await getSecrets();
+  const addresses = await getItems(secretStored.safeAddressesTable);
+  const signers = await getItems(secretStored.safeSignersTable);
+  const schema: Schema = {
+    slackBotToken: secretStored.slackBotToken,
+    slackChannelId: secretStored.slackChannelId,
+    safeAddresses: formatAddress(addresses),
+    safeSigners: formatSigners(signers),
+  };
+
+  return Schema.parse(schema);
+}
+
+function formatAddress(
```

Comment:
This doesn't match the example config. Also, it's kinda the same but different than `formatSigners`. If this were YAML, it would be the same as:

```
safeAddresses:
  - addr_1: alias_1
  - addr_2: alias_2
safeSigners:
  signer_1: alias_1
  signer_2: alias_2
```

Then `safeAddress` is a list of dictionaries, while `safeSigners` is a dictionary. I don't know the whole context, but I feel confident saying that there is no point in `safeAddress` being a list of dictionaries. In particular, the constructor for SafeWatcher only ever looks at the first entry of the dictionary. (bonus points: what is the first entry of a dictionary? https://stackoverflow.com/a/30919039).

You're probably better with a single `dictFromEntries`, with the same code as `formatSigners`.